src/components/create-guild/templates.ts

```diff
@@ -23,6 +23,9 @@ export const TEMPLATES: Array<RoleFormType> = [
     requirements: [
       {
         type: "CAPTCHA",
+        data: {
+          maxAmount: 864000000,
```

Comment:
I think this can be an empty object (so the requirement will be to complete a captcha, without a time limit) 👍

Reply:
In that case, It seems the user is verified instantly without a captcha. It is an expected behavior? 

![image](https://github.com/agoraxyz/guild.xyz/assets/139131231/cc2cf4ff-0202-4e36-8b1e-0ee0bf979527)

Reply:
If you've already completed any captcha on guild, it is expected, yes ;)